lib/http_clients/creditas/person_deletion_api/person_deletion.ex

```diff
@@ -0,0 +1,13 @@
+defmodule HttpClients.Creditas.PersonDeletionApi.PersonDeletion do
+  @moduledoc false
+
+  @type t :: %__MODULE__{
+          id: binary(),
+          person_id: binary(),
```

Comment:
```suggestion
          id: String.t(),
          person_id: String.t(),
```
ref: [binaries](https://elixir-lang.org/getting-started/binaries-strings-and-char-lists.html#binaries)
acho q é menos genérico.. não sei na real o que seria melhor nesse caso 🤔

Reply:
feito!